frontend/src/components/Oracles.js

```diff
@@ -22,16 +20,11 @@ const Oracles = () => {
 		});
 		await updateItems('oracles', updatedList);
 		setOraclesList(updatedList);
+		oracle.retriever && history.push(`/retriever/${oracle.retriever}`, oracle);
```

Comment:
I don't see where this flag is set for the oracle object provided. You can just call `history.push` with no need to check this flag